scripts/invoke_view_function.ts

```diff
@@ -0,0 +1,21 @@
+import { parseArgs, getBatchExchange } from "./util";
+import { factory } from "../src/logging";
+const log = factory.getLogger("scripts.invoke_view_function");
+
+module.exports = async (callback: Truffle.ScriptCallback) => {
+  try {
+    const args = parseArgs();
+    if (args.length < 1) {
+      callback(
+        "Error: This script requires arguments - <functionName> [..args]",
+      );
+    }
+    const [functionName, ...arg] = args;
+    const exchange = await getBatchExchange(artifacts);
+    const info = await exchange.contract.methods[functionName].call(...arg);
```

Comment:
I think this change is responsible for the different return values.
```suggestion
    const info = await (exchange as any)[functionName].call(...arg);
```

I think here you need to explicitly circumvent the type-system.